### ISSUE TITLE
Adhar verification for workers 

### ISSUE THREAD

**Issue by KaizenGirl1111** [KaizenGirl1111/GSGrihSangini#73]
**Lack of verified workers**
The nationality, credibility of a worker can't be gauged without adhar verification[...]

**what can we do about it**
After worker login, the worker can land upto a page where he/she'd require to upload their adhar card image. 

**Alternative **
This is a really hard method, but if you can send an OTP to the adhar card verified no., it'd be great. If you can't, opt for 1st method.

**Any other simpler way?**
You can think of a simpler way to do this.

**Comment by mayank-del** [comment]
@KaizenGirl1111  maam, please assign me in this issue under gssoc 22

**Comment by KaizenGirl1111** [comment]
@mayank-del could you please confirm if you got my request on discord?

**Comment by mayank-del** [comment]
> @mayank-del could you please confirm if you got my request on discord?

@KaizenGirl1111  , can we meet on discord at 9pm

**Comment by mayank-del** [comment]
@KaizenGirl1111  maam, just tell me , should I work on adhar verification with or without previos PR's code???

**Comment by KaizenGirl1111** [comment]
Yes @mayank-del , please.

**Comment by Siddhesh777** [comment]
@mayank-del 
I would like to request you to "fetch and merge" and the "git pull" to your repo..
I think your repo is lagging behind with commits

**Comment by mayank-del** [comment]
> @mayank-del I would like to request you to "fetch and merge" and the "git pull" to your repo.. I think your repo is lagging behind with commits

It means the problem is not due to my code ,ok thank god.

@KaizenGirl1111  maam, So it would be better, if  I add adhaar verification between Login and client request page .

**Comment by mayank-del** [comment]
> > @mayank-del I would like to request you to "fetch and merge" and the "git pull" to your repo.. I think your repo is lagging behind with commits
> 
> It means the problem is not due to my code ,ok thank god.
> 
> @KaizenGirl1111 maam, So it would be better, if I add adhaar verification between Login and client request page .

Means , I am saying that user will login ,then he will verify aadhar , then he will get access to view client's request.

**Comment by KaizenGirl1111** [comment]
Yes the otp option is good.
Also of service is paid, I need to get it in 1-2 days right?

**Comment by KaizenGirl1111** [comment]
@mayank-del

**Comment by mayank-del** [comment]
> Yes the otp option is good. Also of service is paid, I need to get it in 1-2 days right?

Yes maam, It takes 1-2 days to those companies to contact you, then the further process happens.

**Comment by KaizenGirl1111** [comment]
Hey @mayank-del 👋
It is 1 day to go for GSSoC to end, it's okay if we can't add this funtionality in ideal way, we can add a portal for people to upload the adhar image.

If even that is not possible in this short span, it is alright we'll close the issue 

Great working with you, :)

**Comment by mayank-del** [comment]
@KaizenGirl1111  ,No maam, It's mayank here, I found an awsome api

**Comment by mayank-del** [comment]
> @KaizenGirl1111 ,No maam, It's mayank here,I never left any work unfinished.
Actually, I found an awsome api,

it's policy is also very awesome ,I will just add the code to call that api and entire code how to make verify otp and everything today itself.
Only UI would left
